docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+# Deploying a production network
+
+This deployment guide is a high level overview at the proper sequence for setting up a functioning Fabric network, in addition to best practices and a few of the many considerations to keep in mind when deploying. Note that this topic will discuss "setting up the network" as a holistic process from the perspective of a single individual. More likely than not, real world production networks will not be set up by a single individual but as a collaborative effort directed by several individuals (a collection of banks each setting up their own components, for example) instead.
```

Comment:
..overview of...

Reply:
like the 'holistic approach' :-)

Reply:
maybe worth choosing a different word than difficult?  

detailed? complex? maybe.

Reply:
I try not to mince words but "complex" also works for me.